tests/ApplePayGatewayTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Omnipay\Vindicia;
+
+use Omnipay\Vindicia\TestFramework\DataFaker;
+use Omnipay\Tests\GatewayTestCase;
+use Omnipay\Omnipay;
+
+class ApplePayGatewayTest extends GatewayTestCase
+{
+    /**
+     * @return void
+     */
+    public function setUp()
+    {
+        $this->gateway = new ApplePayGateway($this->getHttpClient(), $this->getHttpRequest());
+        $this->gateway->setTestMode(true);
+        $this->faker = new DataFaker();
+
+        $this->pemCertPath = $this->faker->path();
+        $this->keyCertPath = $this->faker->path();
+        $this->keyCertPassword = $this->faker->password();
+
+        $this->validationUrl = $this->faker->url();
+        $this->merchantIdentifier = $this->faker->transactionId();
+        $this->displayName = $this->faker->username();
+        $this->applicationUrl = $this->faker->url();
+    }
+
+    /**
+     * @return void
+     */
+    public function testGetName()
+    {
+        $this->assertSame('Vindicia ApplePay', $this->gateway->getName());
```

Comment:
Once you make this string into a constant in the corresponding class, you can update this to use that.

Reply:
See comment above ^^. For this test, I believe it should be hardcoded, to make sure I'm getting exactly what I'm expecting. If there was a typo in the gateway class that this tests, this would catch that.